src/components/PrioSelector.vue

```diff
@@ -0,0 +1,80 @@
+<template>
+  <select
+    class="todo-priority high"
+    v-on:change="handlePriorityChange"
```

Comment:
In vue3 you don't need to use explicitly v-on before event handlers it is replaced whit "@" so you can simply use `@change`